### DISCUSSION TITLE
Collision Trends in Seattle Examined by Change Over Time

### DISCUSSION THREAD

**Discussion by seanhy1** [jakobzhao/geog495#49] [Final Project]
Link: https://seanhy1.github.io/final_project/geog495_final_project/index.html
GitHub: https://github.com/seanhy1/final_project
Description: This project is an interactive web map featuring the Greater Seattle area, as well as data on every motor vehicle collision collected by the Seattle Police Department and Traffic Records. As a Washingtonian who has resided outside of Seattle until attending the University of Washington, I have grown a stronger understanding of the lack of proper driving that happens within Seattle. Although this application does not necessarily give us a strong indication as to why such association between poor driving capabilities and vehicles within the Greater Seattle area exist, it was worth taking a deeper look into by being able to filter collision data through the aspects of change over time. On a more personal scale, this project carries significance to me as recently one of my closest family friends has dealt with paralysis in their lower body from a car crash that occurred two years ago near UW’s campus. Given a platform to engage in an interactive web application, I wanted to honor them and dedicate my studies to prevent such grievances/pain for locals around the area in hopes that such work can ultimately be examined to best prepare drivers and their safety when putting themselves out on the road where many uncontrollable factors can lead to a potential collision. For this exact reason, this is also why the variable be observed over change in time is specifically injuries suffered through motor vehicle collisions.
<img width="1438" alt="seattle_collisions" src="https://user-images.githubusercontent.com/86577552/145395462-8f155fb2-99b2-49c3-9699-d8705a825265.png">

**Comment by zhuxiaoyu1019** [comment]
I think this is a very well-done data visualization application. I like the color scheme you chose for your base map and data layer, which clearly shows the incident and helps users understand what is happening. A side interactive toolbar makes your app more engaging to users, resulting in them being more likely to recommend and use your app.
In terms of understanding the map, it might be more helpful if the legend specified the type of unit.

**Comment by jacobf19** [comment]
Hi Sean,

I really like the style of this map, its very pleasant to look at and the colors work together well. One suggestion I might have is more ability to filter the data or just less data if filtering is not an option. The map can feel a bit aggressive with the sheer amount of data so it might be helpful to filter it a bit more or just limit the data slightly so its easier to see more individual instances.

**Comment by notzulkifli** [comment]
I enjoy of the slider and how moving the sliders will effect what is being displayed. Amazing use of data sorting and analysis. One thing i would add to this to make it a little more interesting is when you hover over certain areas of the map feature, it could display number of collisions? I know your legend already does a great job in providing the necessary information, but just a minor suggestion for an overall interactive experience.

**Comment by manhongg** [comment]
I thought that your use of color to convey the data and your comprehensive usage of various data types in your map was very effective and made your project interesting to play around with. You did a great job of filtering the data and creating a web app that's user friendly and informative! Although it's not necessarily needed for your web app, an idea to expand on the user interactivity might be to add popups with more information on specific data points.

**Comment by Subarashiihibi** [comment]
I love the visualization of the map. Through the map I can know the seriousness of the collisions clearly. One minor problem could be the color of side bar seems to be too bright and it could be kind of hard to read what is inside the box, and it could be distracting since the color seems to be similar to your markers' color.

**Comment by mhan799** [comment]
I think this is a well-organized and informational map that is also very useful for Seattle residents. The sidebar is also very user-friendly. I think the only thing that could improve is that the base map layer looked a bit simple and does not deliver much information about the location of the collision places.

**Comment by yodapancake** [comment]
I really enjoy using your map. The webpage is clean, the colors work well for the context, and there's freedom to manipulate the data in interesting ways with the side panel. One thing that could be improved is adding the ability to click on the bubbles and see the true number of casualties like with a popup. It's kind of hard to tell what the number of injuries is by just looking at the size of the bubbles/color.
